docs/example_yamls/stellar/star0.yaml

```diff
@@ -0,0 +1,4 @@
+!Star
+position: [2, 3]
```

Comment:
I think it is better to not allow any implicit units, because this will lead to confusion pretty quickly.